src/layouts/default.vue

```diff
@@ -58,6 +58,10 @@ body {
   background-color: $pale-grey;
 }
 
+.app > :nth-child(2) {
```

Comment:
I didn't double check, but I think you can give the `<nuxt />` component this is referencing a class. I abuse nth-child selectors too, but a specific class may be more readable if it's possible.